redash/models/base.py

```diff
@@ -12,10 +12,12 @@
 class RedashSQLAlchemy(SQLAlchemy):
     def apply_driver_hacks(self, app, info, options):
         options.update(json_serializer=json_dumps)
+        options.update(pool_pre_ping=True)
```

Comment:
Are you sure it's needed here too?

Reply:
Yes. Without it the bug is still there. I have tested it.

I will update the pull request with the recommendations later :)